A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -588,9 +588,17 @@ class CDUPerformancePage {
         if (isFinite(mcdu.perfApprDH)) {
             dhCell = mcdu.perfApprDH.toFixed(0);
         }
+        if(mcdu.perfApprDH == -1) {
+            dhCell = "NO";
+        }
         mcdu.onRightInput[2] = () => {
             let value = mcdu.inOut;
             mcdu.clearUserInput();
+            if(value == 'NO' || 'no') {
```

Comment:
Assuming this is plain JavaScript:
```suggestion
            if(value == 'NO' || value == 'no') {
```

Reply:
`value.toLowerCase() === 'no'` would be the choice for many pros. Triple equal sign is the way to go for added explicitness.

Reply:
Most definitely usage of `===` makes most sense. Having no experience with any of development within this project, I'm assuming `mcdu.inOut` represents the value entered through the keyboard by the pilot (or alternatively the value displayed as feedback to the pilot)? If so, then I'm assuming characters are either always lowercase or uppercase and therefore a single equality expression would be enough and no `toLowerCase` would be needed.

Reply:
yes, please confirm if rest of code is checking upper or lowercase, and then do that.